airbyte-cdk/bulk/core/base/src/main/kotlin/io/airbyte/cdk/ssh/TunnelSession.kt

```diff
@@ -67,6 +67,7 @@ fun createTunnelSession(
                     client.connect(sshTunnel.user.trim(), sshTunnel.host.trim(), sshTunnel.port)
                 is SshPasswordAuthTunnelMethod ->
                     client.connect(sshTunnel.user.trim(), sshTunnel.host.trim(), sshTunnel.port)
+                null -> TODO()
```

Comment:
@rodireich @xiaohansong  does this line I added look ok to you?

Reply:
Because this is production code and not test we should define it more clearly.
What is the expectation here - is null an error state? if yes we should throw an error, which is what this `TODO()` does

Reply:
null should not be an error state I think.